cmd/kubeadm/app/cmd/upgrade/diff_test.go

```diff
@@ -41,6 +45,14 @@ func createTestRunDiffFile(contents []byte) (string, error) {
 	return file.Name(), nil
 }
 
+func createRandString(n int) string {
```

Comment:
i think we can avoid using this and just write some generic constant contents to the files instead of random strings.

Reply:
I need some empty file paths as test cases. For the sake of safety, I use this function of generating random strings to generate random paths.

Reply:
i don't think validating contents is needed with random strings.
contents area already tested in TestRunDiff.